Core/Utilities/TimeLeft/TimeLeft.py

```diff
@@ -76,10 +85,23 @@ def getTimeLeft( self, cpuConsumed = 0.0 ):
       return resourceDict
 
     resources = resourceDict['Value']
-    if not resources['CPULimit'] or not resources['WallClockLimit']:
+    self.log.verbose( resources )
+    if not resources['CPULimit'] and not resources['WallClockLimit']:
       # This should never happen
       return S_ERROR( 'No CPU or WallClock limit obtained' )
 
+    # if one of CPULimit or WallClockLimit is missing, compute a reasonable value
+    if not resources['CPULimit']:
+      resources['CPULimit'] = resources['WallClockLimit']
```

Comment:
This would change the behaviour of Timeleft utility I think. I didn't intend to do this with this PR.